asg.tf

```diff
@@ -20,7 +27,7 @@ resource "aws_security_group" "jenkins-master" {
     from_port = 43863
     to_port   = 43863
     protocol  = "tcp"
-    security_groups = ["${aws_security_group.jenkins-agents.id}"]
+    cidr_blocks = ["${var.agents_cidr}"]
```

Comment:
Why a CIDR range over a SG?

Reply:
Because it's possible to enable one or more types of workers. I've got a new way to do it:

`    security_groups = compact([aws_security_group.jenkins-master-lb.id,try(aws_security_group.windows-jenkins-agents[0].id, null),try(aws_security_group.linux-jenkins-agents[0].id, null)])
`
If you are ok with that I will send you a new PR with this and the others fixes.